governance.md

```diff
@@ -79,14 +84,87 @@ process.
 
 ## Elections
 
-The election process is not yet established. It MUST be done by the next election cycle. It MUST
-define, at least, the following:
+> The elections process may become more strict over time, depending on the evolving
+nature of the community.
+
+### Voter Eligibility
+
+The elections are open for anyone to vote. The community asks that voters:
+
+1. Vote only if they are active participants in the community.
+2. Vote only once.
+
+> *Participants* in this context refer to ones that contribute to the community either
+by taking part in community discussions or by writing and reviewing code changes.
+
+### Candidate Eligibility
+
+Candidates must be active participants in the community, and must be nominated by an
+eligible voter.
+
+### Nominations Process
+
+A GitHub issue will be created, calling for nominations through the issue's comments
+section. A link to the issue will be shared (at least) on konflux@googlegroups.com.
+
+Nominations should contain: email address, GitHub handle, company affiliation, and
+Konflux-CI project(s) they participate in, for both the nominee and the nominating
+participants.
+
+One CAN nominate themselves.
+
+Any nominee who accepts the nomination will be on the ballot.
+
+### Election Process
+
+Elections will be held as a poll in Google Forms. voters will select as many candidates
+as there are open seats for. E.g. if the elections are held for replacing 2 open seats,
+then voters will have to select 2 candidates in their vote (a name can only be picked
+once per each voter).
+
+In case the number of seats is greater or equal to the number of candidates,
+voters will also be able to vote against each candidate being elected, and a candidate
+will only be elected if more votes are casted for the candidate than against.
```

Comment:
nitpick: I'd like to avoid this kind of situation to streamline the voting process with only "positive" votes, without "negative" ones. One way of mitigation (not complete elimination) would be to change only half of the committee board every N years, e.g. if a term is 2 years, then exchange half of the board - has the odd number of current board members been a deliberate decision in order to avoid decision ties?

Reply:
The way it was structured originally, one member was set to be reelected after one year (which is due next month) and the other two are due one year later.

We'll only get to vote against if the number of candidates is not bigger than the number of vacancies. This is to make sure that people are not elected only because no one else wanted to be a candidate.

One thing I think we might be missing is for how long we now elect someone for.

Reply:
>One thing I think we might be missing is for how long we now elect someone for.
It's actually mentioned at the top of the document.